src/HW2.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class HW2 {
+
+    public static void homeWorkTask(int a) {
+        a = 555;
```

Comment:
Your method should be general and universal (your method works for 555 only), so we shouldn't write specific values in the middle of a method:

    static void ourMethod(int number) {
        int sum = ((number % 10) + ((number / 10) % 10) + ((number / 100) % 10));
        System.out.println(sum);
    }

Special number (e.g., your variant 555) we should write as an argument to our method when we want to use the method. We use ourMethod in the main method:
ourMethod(555);